code/modules/reagents/reagent_containers.dm

```diff
@@ -67,7 +67,8 @@
 	return
 */
 /obj/item/weapon/reagent_containers/afterattack(obj/target, mob/user , flag)
-	return
+	if(user.a_intent == "harm")
+		return ..()

```

Comment:
I'm pretty sure the way that our click code functions, that `afterattack`'s result is not what causes `attacked_by` to proc - rather, that is in the `attack` of the item, in this case a `grown`. Are you sure this code works as it is? `grown`s on our codebase do not do a parent call if the user is on "harm" intent.